packages/strapi-design-system/src/Button/Button.tsx

```diff
@@ -23,49 +24,28 @@ const LoadingWrapper = styled.div`
   will-change: transform;
 `;
 
-const BoxFullHeight = styled(Box)`
-  height: 100%;
-`;
-
```

Comment:
Assigning a height of 100% actually leads to a problem which is visible e.g. here https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-button--full-width